num_of_buttons: 16 

# Analog Axis
axis_angular: 0
axis_linear: 1
axis_pan: 2
axis_tilt: 3
axis_zoom_wide: 12
axis_zoom_tele: 13

# Axis Scaling
scale_linear: 1.5
scale_angular: 1.5
scale_pan: 90 
scale_tilt: 90 
scale_zoom: 3000 

# Digital Buttons
button_update_ptz: 10
button_dead_man: 11
button_ptz_reset: 2
button_vel_inc: 12
button_vel_dec: 14
button_ptz_vel_inc: 13
button_ptz_vel_dec: 15
